### ISSUE TITLE
This does not work in a virtual enviroment

### ISSUE THREAD

**Issue by evilinsano** [pimoroni/vl53l1x-python#8]
Using this on a Raspberry Pi, and the below instructions fixed the problem I was having. 

Find the file VL53L1X.py in the dist-packages (On RPI3: it's in /usr/local/lib/python3.4/dist-packages/)

Type: 'sudo nano VL53L1X.py' in terminal to edit the file.
Add 'import sys' near the import lines
And replace the lines that say 
```
#Load VL53L1X shared lib ... until pass
```
with
```
# Load VL53L1X shared lib
_POSSIBLE_LIBRARY_LOCATIONS = sys.path
try:
    _POSSIBLE_LIBRARY_LOCATIONS = [os.path.dirname(os.path.realpath(__file__))]$
    _POSSIBLE_LIBRARY_LOCATIONS += [site.getusersitepackages()]
except AttributeError:
    pass
```

**Comment by corymosiman12** [comment]
+1 to this, although was unable to get the suggested solution to work.  Running Python 3.4.3 on stretch and using virtualenv and virtualenvwrapper

**Comment by Gadgetoid** [comment]
Should be fixed in 8e8a29e19c4965219eff5baac085f49502503045 which I've tested with virtualenv and virtualenvwrapper and Python 2.7.9/3.5.3.
